### PR TITLE
Add docs about debugging

### PR DESCRIPTION
**Pull request by lkostrowski**
app-sdk introduced DEBUG integration, this PR adds docs for that

### CODE REVIEW

**Comment by lkostrowski** [comment]
Hey @peelar can you take a look on docs?

File: docs/debugging.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Debugging
+
+Ability to debug is important for apps developers. App SDK provides opt-in verbose logs that can be used
```
Comment (peelar) [inline docs/debugging.md:3]:
```suggestion
The ability to debug is important for app developers. App SDK provides opt-in verbose logs that can be used.
```

Also, the sentence:
> App SDK provides opt-in verbose logs that can be used

reads like it was cut short. Is there supposed to be something after "can be used"? Maybe it needs to be rephrased.

File: docs/debugging.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Debug logs are implemented with [debug](https://www.npmjs.com/package/debug) package and work both in Node and Browser
```
Comment (peelar) [inline docs/debugging.md:5]:
```suggestion
Debug logs are implemented with [debug](https://www.npmjs.com/package/debug) package and work both in Node and Browser.
```

File: docs/debugging.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Enabling verbose logs
+
+Debug logs are aggregated within [namespaces](#namespaces). They can be all enabled, disabled, or confugred.
```
Comment (peelar) [inline docs/debugging.md:9]:
```suggestion
Debug logs are aggregated within [namespaces](#namespaces). They can be enabled, disabled, or configured.
```

File: docs/debugging.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+By default, all verbose logs are disabled
```
Comment (peelar) [inline docs/debugging.md:11]:
```suggestion
By default, all verbose logs are disabled.
```

File: docs/debugging.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+To enable logs, use following:
```
Comment (peelar) [inline docs/debugging.md:13]:
```suggestion
To enable logs, use the following:
```

File: docs/debugging.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+### Enabling in Node context
+
+Add `DEBUG={namespaces}` env variable
```
Comment (peelar) [inline docs/debugging.md:17]:
```suggestion
Add `DEBUG={namespaces}` env variable.
```

File: docs/debugging.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+#### Example
+
+```shell
+DEBUG=* pnpm run dev
+```
+
+For more details check [debug package](https://github.com/debug-js/debug#usage)
```
Comment (peelar) [inline docs/debugging.md:25]:
```suggestion
For more details check [debug package](https://github.com/debug-js/debug#usage).
```

File: docs/debugging.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+### Enabling in Browser context
+
+Set `localStorage.debug = '{namepsaces}'` in your frontend code. For more details see [debug package](https://github.com/debug-js/debug#browser-support)
```
Comment (peelar) [inline docs/debugging.md:29]:
```suggestion
Set `localStorage.debug = '{namepsaces}'` in your frontend code. For more details see [debug package](https://github.com/debug-js/debug#browser-support).
```

File: docs/debugging.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+#### Example
+
+```javascript
+// In devtools console
+localStorage.debug = "*";
+```
+
+## Namespaces
+
+Use namespace name to enable debug logs for each module
```
Comment (peelar) [inline docs/debugging.md:40]:
```suggestion
Use the namespace name to enable debug logs for each module.
```

**Review by peelar — CHANGES_REQUESTED** [review]

File: docs/debugging.md (hunk 1 above)
Comment (lkostrowski) [reply docs/debugging.md]: Rephrased + applied all suggestions

**Review by peelar — APPROVED** [review]

**Review by krzysztofwolski — APPROVED** [review]

File: docs/debugging.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+### Enabling in Browser context
+
+Set `localStorage.debug = '{namepsaces}'` in your frontend code. For more details see [debug package](https://github.com/debug-js/debug#browser-support).
+
+#### Example
+
+```javascript
+// In devtools console
+localStorage.debug = "*";
```
[the remark this answers is not in the document]
Comment (lkostrowski) [reply docs/debugging.md:36]: Good one, will add

**Pull request merged**